src/main/java/bio/terra/service/DrsService.java

```diff
@@ -93,22 +98,35 @@ public DRSObject lookupObjectByDrsId(AuthenticatedUserRequest authUser, String d
         }
     }
 
-    private DRSObject drsObjectFromFSFile(FSFile fsFile, String snapshotId) {
+    private DRSObject drsObjectFromFSFile(FSFile fsFile, String snapshotId, AuthenticatedUserRequest authUser) {
         DRSObject fileObject = makeCommonDrsObject(fsFile, snapshotId);
 
-        DRSAccessURL accessURL = new DRSAccessURL()
+        DRSAccessURL gsAccessURL = new DRSAccessURL()
             .url(fsFile.getGspath());
 
-        DRSAccessMethod accessMethod = new DRSAccessMethod()
+        DRSAccessMethod gsAccessMethod = new DRSAccessMethod()
             .type(DRSAccessMethod.TypeEnum.GS)
-            .accessUrl(accessURL)
+            .accessUrl(gsAccessURL)
             .region(fsFile.getRegion());
 
+        DRSAccessURL httpsAccessURL = new DRSAccessURL()
+            .url(makeHttpsFromGs(fsFile.getGspath()))
+            .headers(makeAuthHeader(authUser));
+
+        DRSAccessMethod httpsAccessMethod = new DRSAccessMethod()
+            .type(DRSAccessMethod.TypeEnum.HTTPS)
+            .accessUrl(httpsAccessURL)
+            .region(fsFile.getRegion());
+
+        List<DRSAccessMethod> accessMethods = new ArrayList<>();
```

Comment:
nit: Can we clean this up to `Arrays.asList(gsAccessMethod, httpAccessMethod)`?

Reply:
Sure. Thanks